Source/Client/Windows/HostWindow.cs

```diff
@@ -104,6 +104,13 @@ public override void DoWindowContents(Rect inRect)
 
             entry = entry.Down(30);
 
+            var appendNameToAutosaveLabel = $"{"MpAppendNameToAutosave".Translate()}:  ";
```

Comment:
So, mostly I like the idea, but I'd really prefer not to clutter the host page with this option. The host game screen should only be for options that make sense on a session by session basis. Is it possible to move this to the mod settings located in the `MultiplayerMod.cs` file?

Reply:
@jacobEAdamson I agree, moving this out of the Host dialog and into Mod settings sounds like a good idea.

Reply:
I'll work on that when I get home later tonight

Reply:
This change has been made.
![autosavePerGameNameModSettings](https://user-images.githubusercontent.com/737914/55660013-324a7580-57d2-11e9-8ac7-d942a547b8b9.png)